modules/mod_oembed/support/oembed_client.erl

```diff
@@ -19,140 +19,78 @@
 -module(oembed_client).
 
 -author("Arjan Scherpenisse <arjan@scherpenisse.net>").
--behaviour(gen_server).
```

Comment:
It was only used to serialize requests, as only the call interface was used.
If you need something parallel or async then it is better to spawn yourself.

Besides that there was an issue where the gen_server was not restarted on failure, as it is not part of a supervisor hierarchy but just a linked process.